.vim/coc-settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I don't think it's normative to include IDE settings in the repo.

Reply:
Vim get thee far from me!